monkey/agent_plugins/exploiters/mssql/src/mssql_exploiter.py

```diff
@@ -96,6 +98,7 @@ def _brute_force_exploit_host(
         download_agent_command: str,
         launch_agent_command: str,
         agent_binary_downloaded: Event,
+        ports_to_try: Set[NetworkPort],
```

Comment:
Try to keep type hints as open as possible. [Postel's Law (the robustness principle)](https://en.wikipedia.org/wiki/Robustness_principle) states "Be conservative in what you do, be liberal in what you accept from others." 

What does this component actually need?
```suggestion
        ports_to_try: Iterable[NetworkPort],
```